### ISSUE TITLE
Ability to disable some of the servers from extensions.

### ISSUE THREAD

**Issue by Uvdon** [recloudstream/cloudstream#1362]
### Describe your suggested feature

The new extension update of code stream and stream play are a mess. I get it, people want 4k + quality but not everyone has the best Internet for that especially when 1080p isn't even bad at all. Therefore, please in the future consider the option for people like us to disable extensions that we don't want.

I know there is a configuration setting in the player, but that never works for some reason. For example if I disable 480p quality it's still going to appear in next movie or episode. Anyways, please and thank you. 

### Acknowledgements

- [X] My suggestion is **NOT** about adding a new provider
- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.

**Comment by C10udburst** [comment]
https://recloudstream.github.io/csdocs/settings/srcpriority/

**Comment by Uvdon** [comment]
That's what I was talking about, it doesn't work at all

**Comment by Moviellect** [comment]
Yeah brother the quality set option doesn't work. Please solve this issue.
